documentation/book/ref-operators-cluster-operator-configuration.adoc

```diff
@@ -28,9 +28,9 @@ The level for printing logging messages. The value can be set to: `ERROR`, `WARN
 `STRIMZI_OPERATION_TIMEOUT_MS`:: Optional, default: 300000 ms. The timeout for internal operations, in milliseconds. This value should be
 increased when using {ProductName} on clusters where regular {ProductPlatformName} operations take longer than usual (because of slow downloading of Docker images, for example).
 
-`STRIMZI_DEFAULT_KAFKA_IMAGE`:: Optional, default `strimzi/kafka:latest`.
-The image name to use as the default when deploying Kafka, if
-no image is specified as the `Kafka.spec.kafka.image` in the xref:assembly-configuring-container-images-deployment-configuration-kafka[].
+`STRIMZI_KAFKA_IMAGE_MAP`:: Required.
+This provides a mapping from Kafka version to the corresponding Docker image containing a Kafka broker of that version. The required syntax is whitespace separated `_<version>_=_<image>_` pairs.
```

Comment:
From the code it seems that not only whitespace is allowed but even comma for example, tab and line feed + carriage return ...

`StringTokenizer tok = new StringTokenizer(str, ", \t\n\r");`

should we mention them as well?

Reply:
I'll mention comma.